commands/ai/ai_commands.py

```diff
@@ -0,0 +1,37 @@
+from discord import app_commands
+import discord
+from modules.ai.llm import get_response, list_models
+from utils.splitter import split_text
+
+def register_commands(
+    tree: discord.app_commands.CommandTree,
+    guilds: list[discord.Object],
+):
+    @tree.command(
+        name="llm",
+        description="Ask the AI a question",
+        guilds=guilds,
+    )
+    @app_commands.describe(question="The question you want to ask the AI")
+    async def llm(interaction: discord.Interaction, question: str, model: str = ""):
```

Comment:
1. I think the list of models already be specified as discord `Choice`s
2. Provide a default one since users probably don't care/know the differences between each model